gd32e10x/standard_peripheral/include/gd32e10x_fmc.c

```diff
@@ -1,880 +0,0 @@
-/*!
```

Comment:
why this commit? just fixup HAL addition commit

Reply:
gd32e10x SoC is already at hal_gigadevice and by unknown reason gd32e10x_fmc.c is duplicated at include dir.
So it is safe drop this file.